AppAuth.podspec

```diff
@@ -55,6 +59,8 @@ It follows the OAuth 2.0 for Native Apps best current practice
 
     # macOS
     externalUserAgent.osx.source_files = "Source/AppAuth/macOS/**/*.{h,m}"
-    externalUserAgent.osx.deployment_target = '10.9'    
+    externalUserAgent.osx.deployment_target = '10.9'
```

Comment:
Delete this line as it appears not needed, and `externalUserAgent.ios.deployment_target`